<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>822</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>822</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="main_page">
     <widget class="QGroupBox" name="gridGroupBox">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>821</width>
        <height>601</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_groups">
         <property name="text">
          <string>Groups</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="pushButton_attendances">
         <property name="text">
          <string>Attendances</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_children">
         <property name="text">
          <string>Children</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_legal_rep">
         <property name="text">
          <string>Legal representative</string>
         </property>
        </widget>
       </item>
       <item row="11" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_employes">
         <property name="text">
          <string>Employes</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pushButton_eventpart">
         <property name="text">
          <string>Event participation</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="pushButton_events">
         <property name="text">
          <string>Events</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QPushButton" name="pushButton_relationship">
         <property name="text">
          <string>Relationship</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="pushButton_activities">
         <property name="text">
          <string>Activities</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="pushButton_resforact">
         <property name="text">
          <string>Responsible for activity</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="pushButton_resforevent">
         <property name="text">
          <string>Responsible for event</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="view_page">
     <widget class="QPushButton" name="pushButtonDelete">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>160</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonUpdate">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>160</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>621</width>
        <height>271</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::TextElideMode::ElideNone</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonCreate">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>88</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="CU_page">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButtonCU">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>420</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>550</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
